edge-modules/TwinTester/TwinEdgeOperationsResultHandler.cs

```diff
@@ -43,27 +47,22 @@ public Task HandleTwinValidationStatusAsync(string status)
 
         public Task HandleReportedPropertyUpdateExceptionAsync(string propertyKey, string failureStatus)
```

Comment:
Since this function no longer needs `string propertyKey ` and ` string failureStatus` please get rid of the input arguments.

Reply:
I removed propertyKey, failureStatus is needed in TwinAllOperationsResultHandler for long haul 

---
In reply to: [361752363](https://github.com/Azure/iotedge/pull/2177#discussion_r361752363) [](ancestors = 361752363)